addons/website/static/tests/tours/edit_link_popover.js

```diff
@@ -184,5 +184,26 @@ wTourUtils.registerWebsitePreviewTour('edit_link_popover', {
         extra_trigger: 'iframe html:has(.o_edit_menu_popover)',
         run: function () {}, // it's a check
     },
+    {
+        content: "Ensure that a click on the link popover link opens a new window",
+        trigger: 'iframe .o_edit_menu_popover a.o_we_url_link[target="_blank"]',
+        extra_trigger: 'iframe .o_edit_menu_popover a.o_we_full_url[target="_blank"]',
+        run: function (actions) {
+            actions.tip_widget.$anchor[0].addEventListener('click', (e) => {
+                const originalPreventDefault = e.preventDefault.bind(e);
+                e.preventDefault = () => {
+                    console.error('clicking on the link should open in a new tab');
+                    document.body.classList.add('o_we_test_error');
+                };
+                originalPreventDefault(); // We do not want to open a new tab in a tour
+            }, {once: true, capture: true});
+            actions.click();
+        },
+    },
+    {
+        content: "Check the previous steps for errors",
+        trigger: 'body:not(.o_we_test_error)',
+        run: function () {},
+    },
```

Comment:
I don't mind it but is this step actually needed? Just curious.

Reply:
I know runbot is red with console.error, but the tour would still display "test successful" in the console, which is why I made this step. But after thinking about it a little bit, I can just throw an error to crash the test, which is what I did now.